Caculator/app/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,6 @@
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
     <color name="blue">#1565C0</color>
-</resources>
\ No newline at end of file
+    <color name="white">#fff</color>
+    <color name="background">#D6D5D5</color>
```

Comment:
Đặt tên theo màu và theo conventions nhé Việt 
```suggestion
    <color name="color_white">#fffff</color>
    <color name="color_alto">#D6D5D5</color>
```
http://chir.ag/projects/name-that-color/#D6D5D5